src/wallet/wallet2.h

```diff
@@ -673,6 +775,11 @@ namespace tools
     uint64_t get_fee_multiplier(uint32_t priority, int fee_algorithm = -1);
     uint64_t get_per_kb_fee();
 
+    crypto::public_key get_multisig_signer_public_key(const crypto::secret_key &spend_skey) const;
+    crypto::public_key get_multisig_signer_public_key() const;
+    crypto::public_key get_multisig_signing_public_key(size_t idx) const;
+    crypto::public_key get_multisig_signing_public_key(const crypto::secret_key &skey) const;
```

Comment:
This function and the other function `get_multisig_signer_public_key(const crypto::secret_key &spend_skey)` don't depend on the `wallet2` member data, so I'd like them to be made `static`. Or alternatively, is it necessary to declare them in the first place, considering that they're being only an alias to `crypto::secret_key_to_public_key`?

Reply:
The intent is that if we change how to determine a signer's "identifier", then it can be done here once and not have to change all the places which call it. Originally, I was using a hash of the key. Now it's a public key. Later, it might turn out we want to salt it or whatever. These functions make sure there's only one place that need changing. Making them static could be done, but they logically belong to the object, not the class, even though they don't use data members, but this is not inherent in the function's intent. A bit different from the previous case, where any wallet can verify any multiisg info's validity, in my opinion.

Reply:
I'm fine then.